diff -uNdr eiskaltdcpp-2.2.6_orig/eiskaltdcpp-qt/src/EiskaltApp.h eiskaltdcpp-2.2.6/eiskaltdcpp-qt/src/EiskaltApp.h
--- eiskaltdcpp-2.2.6_orig/eiskaltdcpp-qt/src/EiskaltApp.h	2012-02-20 23:55:51.000000000 +0400
+++ eiskaltdcpp-2.2.6/eiskaltdcpp-qt/src/EiskaltApp.h	2012-05-12 22:18:05.529171037 +0400
@@ -87,7 +87,7 @@
 class EiskaltApp: public QtSingleCoreApplication {
 Q_OBJECT
 public:
-    EiskaltApp(int argc, char *argv[], const QString& uniqKey): QtSingleCoreApplication(argc, argv, uniqKey)
+    EiskaltApp(int &argc, char *argv[], const QString& uniqKey): QtSingleCoreApplication(argc, argv, uniqKey)
     {
         installEventFilter(&ef);
     }
diff -uNdr eiskaltdcpp-2.2.6_orig/eiskaltdcpp-qt/src/EiskaltApp_haiku.h eiskaltdcpp-2.2.6/eiskaltdcpp-qt/src/EiskaltApp_haiku.h
--- eiskaltdcpp-2.2.6_orig/eiskaltdcpp-qt/src/EiskaltApp_haiku.h	2012-02-20 23:55:51.000000000 +0400
+++ eiskaltdcpp-2.2.6/eiskaltdcpp-qt/src/EiskaltApp_haiku.h	2012-05-12 22:18:16.616565892 +0400
@@ -87,7 +87,7 @@
 class EiskaltApp: public QApplication {
 Q_OBJECT
 public:
-    EiskaltApp(int argc, char *argv[], const QString& uniqKey): QApplication(argc, argv)
+    EiskaltApp(int &argc, char *argv[], const QString& uniqKey): QApplication(argc, argv)
     {
         installEventFilter(&ef);
     }
